source/comply/audit-log.rst

```diff
@@ -68,18 +77,80 @@ The example JSON configuration specifies two log targets: one outputs to the con
       }
     }
 
-Examples of values for the ``AdvancedLoggingConfig`` setting are:
-
-1. Filespec to another configuration file; this file will contain a JSON object
-
-  ``"AdvancedLoggingConfig": "/path/to/audit_log_config.json"``
-
-2. JSON string
-
-  ``"AdvancedLoggingConfig": "{\"sample-console\":{\"type\":\"console\",\"format\":\"plain\",\"format_options\":{\"delim\":\" | \"},\"levels\":[{\"id\":100,\"name\":\"audit-api\"},{\"id\":101,\"name\":\"audit-content\"},{\"id\":102,\"name\":\"audit-permissions\"},{\"id\":103,\"name\":\"audit-cli\"}],\"options\":{\"out\":\"stdout\"},\"maxqueuesize\":1000},\"sample-file\":{\"type\":\"file\",\"format\":\"json\",\"levels\":[{\"id\":100,\"name\":\"audit-api\"},{\"id\":101,\"name\":\"audit-content\"},{\"id\":102,\"name\":\"audit-permissions\"},{\"id\":103,\"name\":\"audit-cli\"}],\"options\":{\"compress\":true,\"filename\":\"audit.log\",\"max_age\":1,\"max_backups\":10,\"max_size\":500},\"maxqueuesize\":1000}}"``
 
-.. note::
-  When using a JSON string as the value of ``AdvancedLoggingConfig``, ensure you escape double quotes (``"``) in the string using a backslash (``\``). You can also use a free online tool, such as `Free Online JSON Escape <https://www.freeformatter.com/json-escape.html>`__ to format the value correctly.
+2. Multi-line JSON in the ``config.json`` file:
+
+  .. code-block:: json
+
+        "AdvancedLoggingJSON": {
+            "file_1": {
+                "Type": "file",
+                "Format": "plain",
+                "Levels": [
+                    {"ID": 5, "Name": "debug", "Stacktrace": false},
+                    {"ID": 4, "Name": "info", "Stacktrace": false},
+                    {"ID": 3, "Name": "warn", "Stacktrace": false},
+                    {"ID": 2, "Name": "error", "Stacktrace": true},
+                    {"ID": 1, "Name": "fatal", "Stacktrace": true},
+                    {"ID": 0, "Name": "panic", "Stacktrace": true}
+                ],
+                "Options": {
+                    "Compress": true,
+                    "Filename": "mattermost_logr.log",
+                    "MaxAgeDays": 1,
+                    "MaxBackups": 10,
+                    "MaxSizeMB": 500 
+                },
+                "MaxQueueSize": 1000
+            }
+        }
+
+3. Environment variable:
+
+  .. code-block:: shell
+
+    export MM_LOGSETTINGS_ADVANCEDLOGGINGJSON=$(jq -n -c '{
```

Comment:
`MM_LOGSETTINGS_ADVANCEDLOGGINGJSON` doesn't apply to audit logging.  That is the config for general logging.  Not sure if having it here is helpful or confusing.  It may be better suited to an equivalent page documenting general logging settings.

Reply:
Thanks, @wiggin77 -- I agree that leaving this section in will introduce confusion, so best to remove. Would you be open to drafting an outline of a general logging docs page where this content would eventually live?